marketplaceItems/avatarCustomization101/appResources/appData/avatarCustomization101.js

```diff
@@ -0,0 +1,937 @@
+//
+//  avatarCustomization101.js
+//
+//  Blueprint App to teach users how to use materials, blendshapes, and flow for their avatars.
+// 
+//  Created by Robin Wilson and Mark Brosche 2/20/2019
+//  Avatar created by Jimi Youm 2/20/2019
+//  Copyright 2019 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+
+/* global GlobalDebugger */
+
+(function () {
+    // Modules
+    Script.include(Script.resolvePath("./resources/modules/flow.js?v12"));
+
+    var AppUi = Script.require("appUi"),
+        URL = Script.resolvePath("./resources/avatarCustomization101_ui.html?v12344555"),
+        CONFIG = Script.require(Script.resolvePath("./resources/config.js?v22222222111")),
```

Comment:
is there a reason these are JS files and not JSONs?

Reply:
Because they have some variables that are used in multiple places or references a .js function (in Materials.js 's case)

Reply:
>they have some variables that are used in multiple places
why does this preclude them from being JSONs?

>references a .js function (in Materials.js 's case)
this makes sense, OK.

Reply:
I'm looking for a better way of doing the below, if you believe I should make the file JSONs, then I will. 

For example, I am avoiding typing "materials" multiple times in the file. I can instead reference STRING_MATERIALS three times in different places in the exported CONFIG object and avoid making a human error within the file.